docs/operator/troubleshooting.md

```diff
@@ -0,0 +1,88 @@
+# Troubleshooting the Starboard Operator
+
+The Starboard operator installs several Kubernetes resources into the `starboard-system` namespace.
+
+Here are the common steps to check whether the operator is running correctly and to troubleshoot common issue.
+
+In addition to having a look at this section, you want to check [previous issues](https://github.com/aquasecurity/starboard/issues) to see if someone from the community had similar problems before.
+Feel free to either [open an issue](https://github.com/aquasecurity/starboard/issues), reach out on [Slack](https://slack.aquasec.com), or post your questions in the [discussion forum.]()
+
+## Installation
+
+Make sure that the latest version of the Starboard Operator is installed inside of your Kubernetes cluster.
+For this, have a look at the installation [options.](./installation)
+
+For instance, if your are using the Helm deployment, you need to check the Helm Chart version deployed to your cluster:
+```
+helm list -n starboard-system
+```
+
+## Starboard Pod not running
+
+The Starboard Operator will run a pod inside the starboard-system namespace. Make sure that the pod is running correctly:
+
+```
+kubectl get pods -n starboard-system
+```
+
+If the pod is in `failed`, `pending`, or `unknown` check the events and the logs of the pod.
```

Comment:
We can use uppercase to incline with K8s docs on https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/
```suggestion
If the pod is in `Failed`, `Pending`, or `Unknown` check the events and the logs of the pod.
```